### DISCUSSION TITLE
BIRT in bugzilla

### DISCUSSION THREAD

**Discussion by colinsutton** [eclipse-birt/birt#889] [Q&A]
<img width="1144" alt="screenshot_592" src="https://user-images.githubusercontent.com/18058891/159576222-031d4c51-554d-489f-8886-065e46cb100f.png">
Should BIRT users be allowed to raise bugs in bugzilla now or should the message be changed to point BIRT users to github?

**Comment by hvbtup** [comment]
AFAIK Wim added a message. But I can't find it.

The old bugs can be found by searching:
![grafik](https://user-images.githubusercontent.com/7644776/159652572-bf7f3aef-ed96-4e69-a877-ce20f2e751d5.png)

IMHO the fact that BIRT is hard to find on bugs.eclipse.org is not a problem.
It was decided to ignore that huge list. It would have been impossible to take care of them anyway.

Those who are interested in BIRT will find be redirected from https://eclipse.org/birt/ to the new website at https://eclipse.github.io/birt-website/. From there, the community page contains a link "[Report a Bug or Enhancement](https://github.com/eclipse/issues)" that links to the GH issues. BTW I fxed that link in the wiki a few minutes ago.

**Reply by hvbtup** [reply]
Correction: I commited a change to the BIRT website (as my PR from January is still stuck). But the wiki still shows the old page.

**Comment by colinsutton** [comment]
But it's not hard to find. Just log in to bugs.eclipse.org and it's listed under runtimes:
<img width="1158" alt="screenshot_593" src="https://user-images.githubusercontent.com/18058891/159840037-31cc551f-0b0b-4f29-bb47-08bfb2b779de.png">

**Comment by hvbtup** [comment]
What I meant is that BIRT is hard to find in [bugs.eclipse.org](https://bugs.eclipse.org/bugs/). But that's _good_ because the team won't look there anyway.

**Reply by colinsutton** [reply]
? 
BIRT is not hard to find.
A BIRT _user_ who finds a bug might go to bugs.eclipse.org and click on 'file a bug'.

**Reply by wimjongman** [reply]
There is no birt category so it is not possible.

**Reply by colinsutton** [reply]
See the screen shot above. It's in the middle.

**Reply by hvbtup** [reply]
Then this is an inconsistency on the bugs.eclipse.org page 😃 . If you choose "browse" you won't find BIRT anymore.

**Reply by wimjongman** [reply]
Correct. This should be filed here: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues

**Reply by TheLaserNinja** [reply]
We migrated all bug reporting over to Github. Even if it exists on Bugzilla, it won't be checked. The link Wim posted above is the correct place to report bugs.

**Reply by colinsutton** [reply]
Hi John, I know the correct place to report bugs, but non-developers won't. We need to fix up the eclipse web pages so that they point to the right place, as in Wim's helpdesk issue.

**Comment by hvbtup** [comment]
Wim has created an issue for this here: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1060
